### PR TITLE
Added WiFi scanner example

### PR DESCRIPTION
**Pull request by YKatrechko**
Added WiFi scanner example by http://forum.m5stack.com/topic/58/lesson-3-wi-fi-scanner

### CODE REVIEW

File: examples/Advanced/home/ekatrechko/WiFi_scanner/home/ekatrechko/WiFi_scanner.ino [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  n = WiFi.scanNetworks();
+  Show();
+}
+
+void setup()
+{
+  M5.begin();
+  WiFi.mode(WIFI_STA);
+  WiFi.disconnect();
+  double vbat = M5.Axp.GetVbatData() * 1.1 / 1000;
```
Comment (guikcd) [inline examples/Advanced/home/ekatrechko/WiFi_scanner/home/ekatrechko/WiFi_scanner.ino:74]: Not really sure that this is required (`vbat` not used in the code below).

Comment (graphics80) [reply examples/Advanced/home/ekatrechko/WiFi_scanner/home/ekatrechko/WiFi_scanner.ino:74]: not used yes.

**Review by graphics80 — CHANGES_REQUESTED** [review]
Pls remove the not used code.

**Comment by Tinyu-Zhao** [comment]
Thank you for your example, but your folder placement does not meet our requirements. Can you place your .ino file like other examples?

**Pull request merged**